### ISSUE TITLE
Panel Did Appear not getting called 

### ISSUE THREAD

**Issue by cnmcgrath** [MatthewYork/MYBlurIntroductionView#22]
When I create a custom panel and have it as the first panel in the array, the method panelDidAppear is not called.

**Comment by bobsmells** [comment]
Yep, same here.  Other panels fire ok, just not the first.

**Comment by bobsmells** [comment]
I managed to fix it by adding the following at the end of the buildScrollViewLeftToRight method.

```
//Trigger the panel did appear method in the delegate
if ([Panels[0] respondsToSelector:@selector(panelDidAppear)]) {
    [Panels[0] panelDidAppear];
}
```

Seems to work ok

**Comment by cnmcgrath** [comment]
Awesome, looks like it worked for me too.

**Comment by MatthewYork** [comment]
I have formally added this fix in 2ab6e0a098970242bb97f188c66e7b4917e11d86 I will work it into a cocoa pod soon

Thanks!

**Comment by bobsmells** [comment]
No worries at all - glad to help and thanks very much for a great
component!!!
